pkg/agent/agent_linux.go

```diff
@@ -35,3 +41,14 @@ func (i *Initializer) prepareOVSBridge() error {
 func (i *Initializer) initHostNetworkFlows() error {
 	return nil
 }
+
+// getTunnelLocalIP returns local_ip of tunnel port.
+// On linux platform, local_ip option is not needed.
+func (i *Initializer) getTunnelPortLocalIP() net.IP {
+	return nil
+}
+
+// createDefaultTunnelPort creates default tunnel port.
+func (i *Initializer) createDefaultTunnelPort(tunnelPortName string) (string, error) {
```

Comment:
I just feel not necessary to add a func just to decide localIP, so suggested to make getTunnelPortLocalIP() and createDefaultTunnelPort() a single func.

But now I see you need to know localIP earlier to check it changes or not. In this case, maybe remove createDefaultTunnelPort(), and always call CreateTunnelPortExt() in agent.go.

Reply:
So @jianjuns , do you think if `getTunnelPortLocalIP()` can be kept for different platform, and call ` CreateTunnelPortExt()` directly in agent.go.

Reply:
Yes, that sounds good to me.

Reply:
Done.